manager/controllers/app/moduleinstance.go

```diff
@@ -311,6 +325,8 @@ func (m *ModuleManager) SelectModuleInstances(item modules.DataInfo, appContext
 		if err != nil {
 			return instances, err
 		}
+		readCluster, vaultAuthMethod, err := readSelector.SelectCluster(item, m.Clusters)
```

Comment:
Same here. Use readCluster to get other properties.

Reply:
OK, tnx